### PR TITLE
fix(flashbag): fixed addFlash in ControllerTrait for Symfony 6

### LINKED ISSUE #535: addFlash function in ControllerTrait not compatible with Symfony 6
**Sylius version affected**: 1.12.*

**Description**  
Using the function addFlash() from the resourceController is not working in Sylius 1.12 / Symfony 6.

The function hasn't been updated like the function addFlash in FlashHelper.

**Steps to reproduce**  
Try to use addFlash() in a controller and to view it.

**Possible Solution**  
Update this function like the one in the FlashHelper

### PR DESCRIPTION
**Pull request by UlrichHP**
| Q               | A
| --------------- | -----
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no 
| Related tickets | fixes #535 
| License         | MIT

### CODE REVIEW

**Review by SVillette — APPROVED** [review]

**Comment by SVillette** [comment]
Hi, friendly ping @loic425 @lchrusciel 

Just checked this code, it's part of Symfony `AbstractController`. Merging this PR would support adding a flash message in the Symfony 6 way.

**Comment by loic425** [comment]
@lchrusciel 
https://github.com/symfony/symfony/blob/6.3/src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php#L176

File: src/Bundle/Controller/ControllerTrait.php [hunk 1]
```diff
@@ -156,13 +158,19 @@ protected function file($file, string $fileName = null, string $disposition = Re
      *
      * @final
      */
-    protected function addFlash(string $type, $message)
+    protected function addFlash(string $type, mixed $message): void
```
Comment (loic425) [inline src/Bundle/Controller/ControllerTrait.php]: Hum, is this change necessary? I think it's a bc-break cause the method is protected.

Comment (lchrusciel) [reply src/Bundle/Controller/ControllerTrait.php]: I have nothing else to add. Other then that, it looks good 👍

**Comment by loic425** [comment]
@UlrichHP I've fixed the CI on 1.10 branch, could you rebase please?

**Comment by UlrichHP** [comment]
@loic425 I just did the rebase. It should be good now :smile:

**Review by loic425 — APPROVED** [review]

**Review by Roshyo — APPROVED** [review]

**Comment by UlrichHP** [comment]
@lchrusciel is this PR still relevant or do you want me to close it ?

**Review by loic425 — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, @UlrichHP!

**Comment by lchrusciel** [comment]
@UlrichHP absolutly! Thanks for the PR. During SyliusDay, I've finally found time for some reviews :)

**Pull request merged**